docs/PID tuning.md

```diff
@@ -1,33 +1,55 @@
+
+Cleanfligth implements three independent filters. The first one directly filters the gyroscopes outputs. The second and third ones filter the P term output and D term input seperatly. By default software filtering is disabled (except for PIDC 5 "Harakiri"). Software filtering is controled by three [CLI parameters](https://github.com/cleanflight/cleanflight/blob/master/docs/Cli.md) which control the three filters available:
 
-**The P term** controls the strength of the correction that is applied to bring the craft toward the target angle or
-rotation rate. If the P term is too low, the craft will be difficult to control as it won't respond quickly enough to
-keep itself stable. If it is set too high, the craft will rapidly oscillate/shake as it continually overshoots its
-target.
+**Attention!** The suggested values are subject to change.
 
-**The I term** corrects small, long term errors. If it is set too low, the craft's attitude will slowly drift. If it is
-set too high, the craft will oscillate (but with slower oscillations than with P being set too high).
+**Attention!** Too low cutoffs will lead to instabillity if PID gains are to high. This is caused by delay induced by the filter.
+* **gyro_cut_hz** controls the cut off frequency of the general gyro output. For a good balanced quad a reasonable setting is 100 or above. Settings can go as low as 20 for bad vibration problems but gains must be reduced significantly (explanation see above).
+* **pterm_cut_hz** filters the output to the P term separately. Reasonable values range down to 20 starting at the gyro value.
+* **dterm_cut_hz** does the same for the input of the D term. The D term is usually the most problematic controller regarding noise. Noise on gyro input is typically high pitched and since the D term generates its output proportional to input change noise gets heavyly amplified. Reasonable values range from 20 to 10.
 
-**The D term** attempts to increase system stability by monitoring the rate of change in the error. If the error is rapidly converging to zero, the D term causes the strength of the correction to be backed off in order to avoid overshooting the target.
+To start tuning one must distinguish oszillation and random agitation. The first is caused by too high PIDs. The second by noise getting to the controllers. A general approach is this gradual guide:
 
+1. Set your PID gains as low as you would fly it normaly.
+2. Set filters to 50/20/10.
+3. Do a test flight and watch your craft carefully. You shoulde immediatly be impressed by how silky smooth level modes are now.
+4. You should now try higher PID gains.
+5. Most likely your craft will get instable pretty soon. You now can try 100/50/20 cutoffs. Oszillations should disappear with these cutoffs.
+6. Watch your craft carfully again and check for noise. If there is any: Lower dterm to 10. If this does not help lower pterm and gyro incrementally (pterm usually half of gyro).
```

Comment:
Here dterm refers to dterm_cut_hz I guess so I would write it explicitly so nobody thinks you mean the D term of the PID. Same for pterm.